paper-input-container.html

```diff
@@ -594,12 +610,21 @@
           if (label) {
             this.$.labelAndInputContainer.style.position = 'relative';
           }
+          if (invalid) {
+            cls += ' is-invalid';
+          }
         }
       } else {
         if (_inputHasContent) {
           cls += ' label-is-hidden';
         }
+        if (invalid) {
```

Comment:
Hmm, does that mean that a `required` empty field starts off invalid? That isn't correct :/